NewsTests/Sceen/Main/NewsViewModelTest.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  NewsViewModelTest.swift
+//  NewsTests
+//
+//  Created by Mohamed Elboraey on 18/05/2025.
+//
+
+import XCTest
+@testable import News
+
+final class NewsViewModelTest: XCTestCase {
+    
+
+    let viewModel = NewsViewModel()
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
```

Comment:
If oyu're using `XCTestCase`, you need to create a freah instance of the view model (System Under Test) each time. Adding the view model as a constant instance variable will be reused and this will cause issues if testing systems that has immutable underlying state that changes during other test cases.